graphql/src/main/java/com/linecorp/armeria/common/graphql/GraphqlUtil.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.graphql;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+import com.linecorp.armeria.server.ServiceRequestContext;
+
+import graphql.GraphQLContext;
+
+/**
+ * Utility for handling the GraphQL.
+ */
+@UnstableApi
+public final class GraphqlUtil {
+
+    private static final String GRAPHQL_CONTEXT_KEY = "com.linecorp.armeria.graphql.context.key";
+
+    /**
+     * Returns a key to put {@link ServiceRequestContext} in {@link GraphQLContext}.
+     * For example:
+     * <pre>{@code
+     * new DataFetcher<>() {
+     *     @Override
+     *     public String get(DataFetchingEnvironment environment) throws Exception {
+     *         final ServiceRequestContext ctx = environment.getGraphQlContext()
+     *                                                      .get(GraphqlUtil.graphqlContextKey());
```

Comment:
Would it be better adding the two getters, moving `GRAPHQL_CONTEXT_KEY` to `InternalGraphqlUtil` and removing `graphqlContextKey()` instead? This way, we can hide the implementation detail (key name).

```
public static ServiceRequestContext serviceRequestContext(DataFetchingEnvironment);
public static ServiceRequestContext serviceRequestContext(GraphQLContext);
```

Reply:
I couldn't do that because graphql-protocol doesn't have a graphql-java dependency. 🤣

Reply:
Could we just remove `graphqlContextKey()` and add getters like the following?
https://github.com/line/armeria/blob/42d86f78fdf38edbca900bebbd6d70c65e245017/brave/src/main/java/com/linecorp/armeria/internal/common/brave/TraceContextUtil.java#L38

Reply:
I think it's a good idea to create `GraphqlUtil.serviceRequestContext(...)` , but I don't understand about removing `graphqlContextKey()` . 😢 We eventually need that method to put the `ServiceRequestContext` in the `GraphqlContext`. To hide this, I think it is to move the package (comon -> server) and change the access modifier for it (public -> default).

Reply:
Should we provide an API like
```java
...
graphQLContext(GraphqlUtil.graphqlContext(ctx))
```

Reply:
Fixed 😄